RNReanimated.podspec

```diff
@@ -1,91 +1,22 @@
-  # /appName/node_modules/react-native/package.json
-  nodeModulesDir = File.join(__dir__, "..", "..", "node_modules")
-  reactNativePackageJson = JSON.parse(File.read(File.join(nodeModulesDir, "react-native", "package.json")))
-  reactNativeVersion = reactNativePackageJson["version"]
-  reactTargetTvOS = reactNativePackageJson["name"] == "react-native-tvos"
-  isUserApp = true
-rescue
-  begin
-    # monorepo
-    # /monorepo/packages/appName/node_modules/react-native-reanimated/RNReanimated.podspec
-    # /monorepo/node_modules/react-native/package.json
-    nodeModulesDir = File.join(__dir__, "..", "..", "..", "..", "node_modules")
-    reactNativePackageJson = JSON.parse(File.read(File.join(nodeModulesDir, "react-native", "package.json")))
-    reactNativeVersion = reactNativePackageJson["version"]
-    reactTargetTvOS = reactNativePackageJson["name"] == "react-native-tvos"
-  rescue
-    begin
-      # Example app in reanimated repo
-      # /react-native-reanimated/RNReanimated.podspec
-      # /react-native-reanimated/{Example,FabricExample,TVOSExample}/node_modules/react-native/package.json
-      if ENV["ReanimatedTVOSExample"] == "1" then
-        appName = "TVOSExample"
-      elsif ENV["RCT_NEW_ARCH_ENABLED"] == "1" then
-        appName = "FabricExample"
-      else
-        appName = "Example"
-      end
-      nodeModulesDir = File.join(__dir__, appName, "node_modules")
-      reactNativePackageJson = JSON.parse(File.read(File.join(nodeModulesDir, "react-native", "package.json")))
-      reactNativeVersion = reactNativePackageJson["version"]
-      reactTargetTvOS = ENV["ReanimatedTVOSExample"] == "1"
-    rescue
-      # should never happen
-      reactNativeVersion = '0.68.0'
-      puts "[RNReanimated] Unable to recognize your `react-native` version! Default `react-native` version: " + reactNativeVersion
-    end
-  end
-end
-
-reactCommonDir = File.join(nodeModulesDir, "react-native", "ReactCommon")
-
-if isUserApp
-  libInstances = %x[find ../../ -name "package.json" | grep "/react-native-reanimated/package.json" | grep -v "/.yarn/"]
-  libInstancesArray = libInstances.split("\n")
-  if libInstancesArray.length() > 1
-    parsedLocation = ''
-    for location in libInstancesArray
-      location['../../'] = '- '
-      location['/package.json'] = ''
-      parsedLocation += location + "\n"
-    end
-    raise "[Reanimated] Multiple versions of Reanimated were detected. Only one instance of react-native-reanimated can be installed in a project. You need to resolve the conflict manually. Check out the documentation: https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/troubleshooting#multiple-versions-of-reanimated-were-detected \n\nConflict between: \n" + parsedLocation
-  end
-end
-
-reactNativeMinorVersion = reactNativeVersion.split('.')[1].to_i
+reanimated_package_json = JSON.parse(File.read(File.join(__dir__, "package.json")))
+rn_info = find_config()
```

Comment:
I think `config` would be a better name for variable here because the function is named `find_config` and it also contains information some about Reanimated.